samples/Compute/enforce-hybrid-use-benefit/azurepolicy.rules.json

```diff
@@ -21,16 +21,77 @@
 				"in": [
 					"2008-R2-SP1",
 					"2008-R2-SP1-smalldisk",
+					"2008-R2-SP1-zhcn",
 					"2012-Datacenter",
+					"2012-datacenter-gensecond",
 					"2012-Datacenter-smalldisk",
+					"2012-datacenter-smalldisk-g2",
+					"2012-Datacenter-zhcn",
+					"2012-datacenter-zhcn-g2",
 					"2012-R2-Datacenter",
+					"2012-r2-datacenter-gensecond",
 					"2012-R2-Datacenter-smalldisk",
+					"2012-r2-datacenter-smalldisk-g2",
+					"2012-R2-Datacenter-zhcn",
+					"2012-r2-datacenter-zhcn-g2",
 					"2016-Datacenter",
+					"2016-datacenter-gensecond",
+					"2016-datacenter-gs",
 					"2016-Datacenter-Server-Core",
+					"2016-datacenter-server-core-g2",
 					"2016-Datacenter-Server-Core-smalldisk",
+					"2016-datacenter-server-core-smalldisk-g2",
 					"2016-Datacenter-smalldisk",
+					"2016-datacenter-smalldisk-g2",
 					"2016-Datacenter-with-Containers",
-					"2016-Datacenter-with-RDSH"
```

Comment:
why was the -rdsh sku removed?

Reply:
I used the following command to list all the Windows Server skus: 

`Get-AzVMImageSku -Location "East US" -PublisherName "MicrosoftWindowsServer" -Offer "WindowsServer" | Select Skus`

That sku is not listed in the export.  Is that still a valid sku?  I can re-add it.